src/mscorlib/shared/System/Globalization/TimeSpanParse.cs

```diff
@@ -1592,160 +1509,158 @@ internal bool TryParse(ReadOnlySpan<char> input, ref TimeSpanResult result)
                         time += remainingTime;
                     }
                 }
+
                 if (negative)
                 {
                     time = -time;
                     // Allow -0 as well
                     if (time > 0)
                     {
-                        result.SetFailure(ParseFailureKind.Overflow, "Overflow_TimeSpanElementTooLarge");
-                        return false;
+                        return result.SetFailure(ParseFailureKind.Overflow, SR.Overflow_TimeSpanElementTooLarge);
```

Comment:
Using `SR.Overflow_TimeSpanElementTooLarge` for the failure case will make the TryXXX methods a lot slower when they return false. This should stay on the CoreCLR plan.

Reply:
`SR.Overflow_TimeSpanElementTooLarge` will fetch the resource string. It is not a cheap operation.

Reply:
Ah, makes sense.  I'll switch this back from the corert approach to the coreclr one.

Reply:
I've fixed this.  Thanks for the catch.